tools/eol_cve/eol-cve.csv

```diff
@@ -0,0 +1,120 @@
+CVE-2018-7164,"9.7.x || 10.x","^9.11.2 || ^10.4.1","4.x || 5.x || 7.x"
+CVE-2018-7167,"6.x || 8.x || 9.x","^6.14.3 || ^8.11.3 || ^9.11.2","4.x || 5.x"
+CVE-2018-7166,10.x,>= 10.9.0,"4.x || 5.x || 7.x || 9.x"
+CVE-2018-12115,<= 10,"^6.14.4 || ^8.11.4 || >= 10.9.0","4.x || 5.x || 7.x || 9.x"
+CVE-2018-12116,"6.x || 8.x","^6.15.0 || ^8.14.0","4.x || 5.x"
+CVE-2018-12120,"6.x || 8.x","^6.15.0 || ^8.14.0","4.x || 5.x"
+CVE-2018-12121,"6.x || 8.x || 10.x || 11.x","^6.15.0 || ^8.14.0 || ^10.14.0 || ^11.3.0","4.x || 5.x"
+CVE-2018-12122,"6.x || 8.x || 10.x || 11.x","^6.15.0 || ^8.14.0 || ^10.14.0 || ^11.3.0","4.x || 5.x"
+CVE-2018-12123,"6.x || 8.x || 10.x || 11.x","^6.15.0 || ^8.14.0 || ^10.14.0 || ^11.3.0","4.x || 5.x"
+CVE-2019-5737,"6.x || 8.x || 10.x || 11.x","^6.17.0 || ^8.15.1 || ^10.15.2 || ^11.10.1","4.x || 5.x"
+CVE-2019-5739,6.x,^6.17.0,"4.x || 5.x"
+CVE-2019-9511,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9512,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9513,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9514,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9515,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9516,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9517,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-9518,"8.x || 10.x || 12.x","^8.16.1 || ^10.16.3 || ^12.8.1","4.x || 5.x || 6.x || 7.x"
+CVE-2019-15604,"10.x || 12.x || 13.x","^10.19.0 || ^12.15.0 || ^13.8.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2019-15605,"10.x || 12.x || 13.x","^10.19.0 || ^12.15.0 || ^13.8.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2019-15606,"10.x || 12.x || 13.x","^10.19.0 || ^12.15.0 || ^13.8.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2020-8201,"12.x || 14.x","^12.18.4 || ^14.11.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x || 11.x"
+CVE-2020-8251,14.x,^14.11.0,"4.x || 5.x || 6.x || 7.x || 8.x || 9.x || 11.x || 13.x"
+CVE-2020-8252,"10.x || 12.x || 14.x","^10.22.1 || ^12.18.4 || ^14.9.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2020-8277," 12.x || 14.x || 15.x"," ^12.19.1 || ^14.15.1 || 15.2.1","4.x || 5.x || 6.x || 7.x || 8.x || 9.x || 11.x"
+CVE-2020-8265,"10.x || 12.x || 14.x || 15.x","^10.23.1 || ^12.20.1 || ^14.15.4 || ^15.5.1","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2020-8287,"10.x || 12.x || 14.x || 15.x","^10.23.1 || ^12.20.1 || ^14.15.4 || ^15.5.1","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2020-1971,"10.x || 12.x || 14.x || 15.x","^10.23.1 || ^12.20.1 || ^14.15.4 || ^15.5.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
+CVE-2021-22883," 10.x || 12.x || 14.x || 15.x"," ^10.24.0 || ^12.21.0 || ^14.16.0 || ^15.10.0","4.x || 5.x || 6.x || 7.x || 8.x || 9.x"
```

Comment:
This is still not accurate. It's missing v11.x, which was EOL in 2019 and this CVE has been issued in 2021-02-23.

Reply:
thats right, the mistake was that I was taking in consideration the oldest release line while I should take the newest and consider everything behind it